### PR TITLE
13.0 mig base comment template

### PR DESCRIPTION
**Pull request by emagdalenaC2i**
Superseed #120

### CODE REVIEW

**Comment by AaronHForgeFlow** [comment]
It seems ci is red because of the pre-commit thing again

**Comment by emagdalenaC2i** [comment]
@aheficent No, this time seems it is a 'proper' issue ;-) 
I will take a look later

File: base_comment_template/views/res_partner.xml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<odoo>
+
+    <record id="view_partner_form" model="ir.ui.view">
+        <field name="model">res.partner</field>
+        <field name="inherit_id" ref="base.view_partner_form"/>
+        <field name="priority" eval="2"/>
+        <field name="arch" type="xml">
+            <group name="sale" position="inside">
+                <field name="property_comment_template_id" attrs="{'invisible': ['|',('customer', '=', False), ('is_company', '=', False), ('parent_id', '!=', False)]}"/>
```
Comment (jimhoefnagels) [inline base_comment_template/views/res_partner.xml]:
```suggestion
                <field name="customer_rank" invisible="1"/>
                <field name="property_comment_template_id" attrs="{'invisible': ['|',('customer_rank', '=', 0), ('is_company', '=', False), ('parent_id', '!=', False)]}"/>
```

File: base_comment_template/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+{
+    "name": "Base Comments Templates",
+    "summary": "Comments templates on documents",
+    "version": "13.0.1.0.0",
+    "category": "Sale",
+    "website": "https://github.com/OCA/account-invoice-reporting",
+    "author": "Camptocamp, Odoo Community Association (OCA)",
+    "license": "AGPL-3",
+    "installable": True,
+    "depends": ["base"],
```
Comment (jimhoefnagels) [inline base_comment_template/__manifest__.py]:
```suggestion
    "depends": ["account"],
```

**Review by jimhoefnagels — CHANGES_REQUESTED** [review]

File: base_comment_template/models/res_partner.py [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+from odoo import api, fields, models
+
+
+class ResPartner(models.Model):
+    _inherit = "res.partner"
+
+    property_comment_template_id = fields.Many2one(
+        comodel_name="base.comment.template",
+        string="Conditions template",
+        company_dependant=True,
```
Comment (pedrobaeza) [inline base_comment_template/models/res_partner.py:12]:
```suggestion
        company_dependent=True,
```
I have already fixed it in v12

**Comment by pedrobaeza** [comment]
Superseded by #149

**Pull request closed without merging**